app/routes/events/view.js

```diff
@@ -9,7 +9,7 @@ export default Route.extend({
 
   model(params) {
     return this.store.findRecord('event', params.event_id, {
-      include: 'event-topic,event-sub-topic,event-type,event-copyright,tax,sponsors'
+      include: 'event-topic,event-sub-topic,event-type,event-copyright,tax,sponsors,speakers-call,microlocations,tracks,session-types'
```

Comment:
This route should only load the event. Nothing more. Any related records to be loaded at the child route that would use it.